### PR TITLE
aziot: tweak `check-list` flags to be consistent with `check`

### PR DESCRIPTION
**Pull request by daprilik**
In response to https://github.com/Azure/iotedge/pull/4153#discussion_r555611080

### CODE REVIEW

File: aziot/src/check_list.rs [hunk 1]
```diff
@@ -8,15 +8,33 @@ use aziot_check_common::CheckListOutput;
...
+            _ => return Err("invalid output format"),
+        })
+    }
 }
 
 pub fn check_list(cfg: CheckListOptions) -> Result<()> {
     let checks = crate::internal::check::all_checks();
 
-    if cfg.json {
+    if matches!(cfg.output, OutputFormat::Json) {
```
Comment (arsing) [inline aziot/src/check_list.rs:37]: Make this a `match` so that it's exhaustive.

Comment (daprilik) [reply aziot/src/check_list.rs:37]: b-b-but think of the extra layer of nesting!

Comment (arsing) [reply aziot/src/check_list.rs:37]: I do. Exhaustiveness is nice.

Comment (daprilik) [reply aziot/src/check_list.rs]: fixed.

**Review by arsing — APPROVED** [review]

**Pull request merged**